### DISCUSSION TITLE
Plex plugin

### DISCUSSION THREAD

**Discussion by tom76017** [GerryDazoo/Slinger#18] [Ideas]
Adding a plugin for Plex would be great!  The original TiVo plugin was deactivated.

**Comment by GerryDazoo** [comment]
I doubt this will happen. I found this

August 2022 Update: Plex is phasing out plugins — especially those that require UI support or play content (like streaming plugins). Plex’s representative couldn’t give me an exact date when plugin support would be removed completely but told me that you should no longer rely on plugins to work. I will keep testing and update this page when I know more.

 

But if you can find an existing plugin that does HTTP video streaming, you should be able to do that.

 

From: tom76017 ***@***.***> 
Sent: Tuesday, August 09, 2022 3:47 PM
To: GerryDazoo/Slinger ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [GerryDazoo/Slinger] Plex plugin (Discussion #18)

 

Adding a plugin for Plex would be great! The original TiVo plugin was deactivated.

**Comment by tom76017** [comment]
Thanks for the reply Gerry,

I'm still running WebTools on my server.

Check out this link.

https://www.purevpn.com/blog/best-plex-plugins/

Tom Meyer

**Comment by tom76017** [comment]
I think you're right.  I couldn't find any plugins that had streaming.

Tom
